ucm2/Intel/sof-hda-dsp/dsp.conf

```diff
@@ -36,6 +36,12 @@ Define {
 	Dmic0CaptureBeamformerBytes "not available"
 	Dmic0CaptureBeamformerBeamSwitch "not available"
 	Dmic0CaptureBeamformerBeamAngleSet "not available"
+        MicArrayCaptureIirBytes "not available"
```

Comment:
Note: The Dmic0Capture could be replaced by MicArrayCapture definitions to make more compact code. Today I would say it's very unlikely to have a device with both types of internal microphones (PCH-DMIC and SDW-MIC). This version would handle both if such would ever happen.